packages/components/src/components/modal/modal.jsx

```diff
@@ -39,11 +39,11 @@ class ModalElement extends React.PureComponent {
 
     handleClickOutside = event => {
         const path = event.path || (event.composedPath && event.composedPath());
-
+        const [blockly_injection_div] = document.querySelectorAll('.injectionDiv');
```

Comment:
We shouldn't add logic specific to a package (in this case `@deriv/bot-web-ui`) consuming `@deriv/components`. We should find a different approach.

Reply:
In that case, we could add a props to modal to handle that.